diff --git a/node_modules/react-native-enriched-markdown/ios/styles/StyleConfig.mm b/node_modules/react-native-enriched-markdown/ios/styles/StyleConfig.mm
index 964ca1d..dce8705 100644
--- a/node_modules/react-native-enriched-markdown/ios/styles/StyleConfig.mm
+++ b/node_modules/react-native-enriched-markdown/ios/styles/StyleConfig.mm
@@ -2109,7 +2109,7 @@ static const CGFloat kDefaultMinGap = 4.0;
     _tableHeaderFont = [RCTFont updateFont:nil
                                 withFamily:family
                                       size:@(_tableFontSize)
-                                    weight:normalizedFontWeight(@"bold")
+                                    weight:nil
                                      style:nil
                                    variant:nil
                            scaleMultiplier:[self effectiveScaleMultiplierForFontSize:_tableFontSize]];
